manager/README.md

```diff
@@ -0,0 +1,54 @@
+# manager Project
```

Comment:
What about remove ~all the content of this file? This is just the default Quarkus readme that is not really necessary for us. A generic 
```suggestion
# manager Project

WIP
```
Should be enough. Wdyt?